support-workers/src/main/scala/com/gu/support/workers/lambdas/SendAcquisitionEvent.scala

```diff
@@ -54,6 +63,13 @@ object SendAcquisitionEvent {
     override def getMessage: String = errors.map(_.getMessage).mkString(". ")
   }
 
+  def paymentProviderFromPaymentMethod(paymentMethod: PaymentMethod): thrift.PaymentProvider =
```

Comment:
interesting that we are using thrift defintions for cloudwatch alarms. However at least it would set off the alarm if someone changed the thrift definition.